.github/issue_template.md

```diff
@@ -0,0 +1,35 @@
+# Describe the issue
+
+Thank you for contributing to the project! Please take some time to fill out this template, to help us quickly and smoothly handle your PR.
+
+## Summary
+
+(Briefly describe the issue encountered.)
+
+## Steps to Reproduce
+
+(Provide a clear and concise list of steps to reproduce the issue.)
+
+1. Step 1
+2. Step 2
+3. etc.
+
+## Environment
+
+- Browser: (e.g., Chrome, Firefox)
+- Operating System: (e.g., Windows, macOS, Linux)
+- Version of the design system: (e.g., v1.2.3)
+
+## Checklist
+
+Before submitting your issue, please ensure you have done the following:
+
+- [ ] Searched the issue tracker to ensure the issue has not already been reported.
+- [ ] Verified that the issue can be consistently reproduced.
+- [ ] Checked if the issue is already documented.
+
+## Additional Notes
+
+- Mention if the issue is blocking development or critical for release.
```

Comment:
Will someone ever say it doesn’t? :)

Reply:
No, probably not. Shall we remove this point?

Reply:
Yes, let’s.